package.json

```diff
@@ -60,6 +60,7 @@
     "@storybook/builder-webpack5": "^6.4.19",
     "@storybook/manager-webpack5": "^6.4.19",
     "@storybook/react": "^6.4.19",
+    "@types/autosize": "^4.0.1",
```

Comment:
Please re-run `npm install` - it appears that the package-lock.json is not correctly updated.